Frontend/ui-driver/src/Screens/ReferralScreen/Controller.purs

```diff
@@ -286,4 +315,34 @@ transformLeaderBoard (DriversInfo driversInfo) isMaskedName = {
   , profileUrl : Nothing
   , rank : driversInfo.rank
   , rides : driversInfo.totalRides
-}
\ No newline at end of file
+}
+
+getReferralStage :: Merchant -> ReferralType
+getReferralStage merchant =
+  case getValueFromConfig "referralType" of
+    "LeaderBoard" -> LeaderBoard
+    "QRScreen" -> QRScreen
+    _ -> LeaderBoard
+
+getPlayStoreLink :: Merchant -> String
+getPlayStoreLink merchant =
+  case merchant of
+    NAMMAYATRI -> "https://play.google.com/store/apps/details?id=in.juspay.nammayatri"
+    YATRI -> "https://play.google.com/store/apps/details?id=net.openkochi.yatri"
+    YATRISATHI -> "https://nammayatri.in/link/rider/gnop"
+    _ -> "https://play.google.com/store/apps/details?id=in.juspay.nammayatri"
```

Comment:
`getAppLink` function already exists, we can modify that as per need and move that function to `Utils`